database/fake_data.sql

```diff
@@ -0,0 +1,295 @@
+('ASTRO_star', 'https://dq0hsqwjhea1.cloudfront.net/Interactive-Sky-Chart-600px.jpg', '2024-10-14', 14, 14),
+('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-15', 15, 15),
+('ASTRO_star', 'https://dq0hsqwjhea1.cloudfront.net/Interactive-Sky-Chart-600px.jpg', '2024-10-16', 3, 25),
+('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-17', 5, 12),
+('ASTRO_moon', 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/FullMoon2010.jpg/1200px-FullMoon2010.jpg', '2024-10-18', 7, 45),
+('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-19', 10, 88),
+('ASTRO_star', 'https://dq0hsqwjhea1.cloudfront.net/Interactive-Sky-Chart-600px.jpg', '2024-10-20', 12, 36),
+('ASTRO_moon', 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/FullMoon2010.jpg/1200px-FullMoon2010.jpg', '2024-10-21', 15, 55),
+('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-22', 20, 8),
+('ASTRO_star', 'https://dq0hsqwjhea1.cloudfront.net/Interactive-Sky-Chart-600px.jpg', '2024-10-23', 25, 73),
+('ASTRO_moon', 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/FullMoon2010.jpg/1200px-FullMoon2010.jpg', '2024-10-24', 30, 21),
+('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-25', 35, 50);
+
+-- NASA_apod: https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg
+-- ASTRO_moon: https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/FullMoon2010.jpg/1200px-FullMoon2010.jpg
+-- ASTRO_star: https://dq0hsqwjhea1.cloudfront.net/Interactive-Sky-Chart-600px.jpg
+
+
+INSERT INTO subscriber (subscriber_username, subscriber_phone, subscriber_email)
+VALUES
+('johndoe', '+12345678901', 'johndoe@example.com'),
+('janedoe', '+12345678902', 'janedoe@example.com'),
+('samsmith', '+12345678903', 'samsmith@example.com'),
+('alicewong', '+12345678904', 'alicewong@example.com'),
+('michaelbrown', '+12345678905', 'michaelbrown@example.com'),
+('emilydavis', '+12345678906', 'emilydavis@example.com'),
+('davidjohnson', '+12345678907', 'davidjohnson@example.com'),
+('sarahwilson', '+12345678908', 'sarahwilson@example.com'),
+('chrislee', '+12345678909', 'chrislee@example.com'),
+('lauragarcia', '+12345678910', 'lauragarcia@example.com')
+('robertmartin', '+12345678911', 'robertmartin@example.com'),
+('lisawalker', '+12345678912', 'lisawalker@example.com'),
+('tommyadams', '+12345678913', 'tommyadams@example.com'),
+('nataliebell', '+12345678914', 'nataliebell@example.com'),
+('jacobwhite', '+12345678915', 'jacobwhite@example.com'),
+('oliviayoung', '+12345678916', 'oliviayoung@example.com'),
+('ethanturner', '+12345678917', 'ethanturner@example.com'),
+('gracehall', '+12345678918', 'gracehall@example.com'),
```

Comment:
solid emails and numbers

Reply:
Indubitably

Reply:
Nice example of test emails. We can't test our SNS system works (actually sends messages) with this of course, but we can use them for ensuring our database is functioning as intended.